src/pages/community/people.md

```diff
@@ -0,0 +1,104 @@
+---
+id: people
+title: Who We Are
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Project Members
+
+YuniKorn community is open, diverse, and collaborative. It is not owned
+by any entity. Project members share the responsibility to manage, and maintain the project.
+People in the community includes PPMC ([Podling Project Management Committee](https://incubator.apache.org/guides/ppmc.html))
```

Comment:
```suggestion
People in the community include Podling Project Management Committee ([PPMC](https://incubator.apache.org/guides/ppmc.html))
```

Reply:
It's a convention to put the acronyms in parentheses after the full expanded name

Reply:
Fixed, thanks